subscriptions/tests.py

```diff
@@ -3060,6 +3060,200 @@ def test_filter_by_messageset(self):
+        """
+        stdout, stderr = StringIO(), StringIO()
+
+        registration = {
+            "id": "8646b7bc-b511-4965-a90b-1111111111111",
+            "data": {
+                "exclude_report": True
+            }
+        }
+
+        # mock registration lookup
+        responses.add(
+            responses.GET,
+            "http://seed-hub/api/v1/registrations/?mother_id=8646b7bc-b511-4965-a90b-e1145e398703",  # noqa
+            json={
+                "count": 1,
+                "next": None,
+                "previous": None,
+                "results": [registration]
+            },
+            status=200, content_type='application/json',
+            match_querystring=True
+        )
+
+        identity = {
+            "id": "8646b7bc-b511-4965-a90b-e1145e398703",
+            "version": 1,
+            "details": {
+                "default_addr_type": "msisdn",
+                "addresses": {
+                    "msisdn": {
+                        "+2345059992222": {}
+                    }
+                },
+                "receiver_role": "mother",
+                "linked_to": None,
+                "preferred_msg_type": "text",
+                "preferred_language": "eng_ZA",
+                "exclude_report": True
+            },
+            "created_at": "2015-07-10T06:13:29.693272Z",
+            "updated_at": "2015-07-10T06:13:29.693298Z"
+        }
+
+        # mock identity lookup
+        responses.add(
+            responses.GET,
+            "http://seed-identity-store/api/v1/identities/8646b7bc-b511-4965-a90b-e1145e398703/",  # noqa
+            json=identity,
+            status=200, content_type='application/json',
+            match_querystring=True
+        )
+
+        # Sub that should be updated
+        sub = self.make_subscription()
+        sub.process_status = 5
+        sub.save()
+
+        # Sub that should not be
+        self.make_subscription()
+
+        call_command('mark_invalid_subscriptions', '--hub-url',
+                     'http://seed-hub/api/v1/', '--hub-token', 'HUBTOKEN',
+                     stdout=stdout, stderr=stderr)
+
+        output = stdout.getvalue().strip()
+        self.assertEqual(output, "Updated 0 identities and 0 registrations.")
+
+    @responses.activate
+    def test_mark_invalid_subscription_noop(self):
+        """
+        If there are no subscription with process_astatus of 5 nothing
+        should be updated
+        """
+        stdout, stderr = StringIO(), StringIO()
+
+        self.make_subscription()
+
+        call_command('mark_invalid_subscriptions', '--hub-url',
+                     'http://seed-hub/api/v1/', '--hub-token', 'HUBTOKEN',
+                     stdout=stdout, stderr=stderr)
+
+        output = stdout.getvalue().strip()
+        self.assertEqual(output, "Updated 0 identities and 0 registrations.")
+
+    @responses.activate
+    def test_mark_invalid_subscription_invalid(self):
```

Comment:
maybe `test_mark_invalid_subscription_missing_args` would be a clearer method name?